cmd/lakom/app/app.go

```diff
@@ -211,6 +215,7 @@ func (o *Options) Run(ctx context.Context) error {
 		WithCacheTTL(o.CacheTTL).
 		WithCacheRefreshInterval(o.CacheRefreshInterval).
 		WithUseOnlyImagePullSecrets(o.UseOnlyImagePullSecrets).
+                WithAllowInsecureRegistries(o.AllowInsecureRegistries).
```

Comment:
`cosignSignatureVerifyHandler` must implement the same feature because it is also communicating with the same OCI registry.